src/osd/PGLog.h

```diff
@@ -1475,6 +1475,9 @@ struct PGLog : DoutPrefixProvider {
 	    ceph_assert(dups.back().version < dup.version);
 	  }
 	  dups.push_back(dup);
+	  if (dups.size() >= g_conf()->osd_pg_log_dups_tracked) {
```

Comment:
make check is failing with https://jenkins.ceph.com/job/ceph-pull-requests/94203/consoleFull#-1643918307e4bfde06-44c1-4b3c-8379-d9ee175fb257

Reply:
fixed